docs/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with python 3.10
 # To update, run:
 #
-#    pip-compile
+#    pip-compile --extra=doc --output-file=docs/requirements.txt pyproject.toml
```

Comment:
I was curious if this was possible. This is a much nicer way to run pip-tools :+1: